docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -9,80 +9,151 @@ track: "USERS › APPLICATION DEVELOPER › FOUNDATIONAL"
+
+If you're not running Kubernetes or a similar automated system, you might find the following scenario familiar:
+1. One instance of your app (a complete machine instance or just a container) goes down.
+2. Because your team has monitoring set up, this pages the person on-call.
+3. The on-call has to go in, investigate, and manually spin up a new instance.
+4. Depending how your team handles DNS/networking, the on-call may also need to also update the service discovery mechanism to point at the IP of the new Rails instance rather than the old.
+
+This process can be tedious and also inconvenient, especially if (2) happens in the early hours of the morning!
+
+**If you have Kubernetes set up, however, manual intervention is not as necessary.** Kubernetes will gracefully handle (3) and (4) on your behalf. As a a result, it's often referred to as a *self-healing* system.
+
+On a high-level, there are two key concepts to making this work: the *Kubernetes API* and the *Control Plane*.
+
+#### Kubernetes API
+
+For Kubernetes to be useful, it needs to know *what* sort of cluster state you want it to maintain. The {% glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" %} provides a way for you to declare this desired state, in terms of API objects (such as {% glossary_tooltip text="Deployments" term_id="deployment" %}). Cluster state includes but is not limited to the following information:
 
 * The applications or other workloads to run
 * The container images for your applications and workloads
 * Allocation of network and disk resources
 
-Once you’ve set your desired state -- that is, applied your manifest or spec to your cluster -- the Kubernetes Control Plane works to make the cluster’s current state match the desired state. To do so, Kubernetes performs a variety of tasks automatically: starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster:
+Each API object can be described in a YAML or JSON file called a *manifest*, which can be submitted to the appropriate API server endpoint.
+
+`kubectl`, the command-line tool that allows you to easily read or modify your Kubernetes cluster, is basically just a user-friendly wrapper for the Kubernetes API. You're able to scale app instances and get node info because, under the hood, it's making API requests for you.
+
+You can read more about the Kubernetes API [here](/docs/concepts/overview/working-with-objects/kubernetes-objects/){:target="_blank"}.
+
+#### Control Plane
+
+Once you’ve set your desired state -- that is, applied your YAML manifest to your cluster -- the **Kubernetes Control Plane** works to make the cluster’s current state match the desired state.
+
```

Comment:
Misleading, I think. I don't set my desired state. I declare my desired state. Even after my YAML files are applied to the cluster, the cluster might not be in my desired state.

Suggestion:
Your YAML manifests declare your desired state. After you apply your YAML manifests to your cluster, the Kubernetes control plane works to make the cluster's current state match the desired state.